optimizer/src/main/java/com/netease/arctic/optimizer/flink/FlinkOptimizer.java

```diff
@@ -78,16 +78,16 @@ public void start() throws IOException {
     }
     String cmd = flinkHome + "/bin/flink run -m yarn-cluster ";
 
-    //add flink executor config
+    // add flink executor config
     int tmMemory = groupProperties.getInteger("taskmanager.memory");
     cmd += " -ytm " + tmMemory;
     int jmMemory = groupProperties.getInteger("jobmanager.memory");
     cmd += " -yjm " + jmMemory;
 
     // spill map config
-    Boolean enableSpillMap = groupProperties.getBoolean("enable_spill_map");
-    String backendBaseDir = groupProperties.getString("backend_base_dir");
-    Long maxDeleteMemorySize = groupProperties.getLong("max_delete_memory_size_in_byte");
+    Boolean enableSpillMap = groupProperties.getBoolean("spillable.map.enabled");
```

Comment:
Should we manage these property names in a static class?

Reply:
Yes